tests/src/runonly.targets

```diff
@@ -0,0 +1,13 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="CreateManifestResourceNames" />
+  <Target Name="Compile" />
+  <Target Name="CoreCompile" />
+
+  <!-- if this is a RunOnly project that has been scheduled to build, make the output directory and build any dependent projects -->
+  <Target Name="Build" Condition="('$(CLRTestKind)'=='RunOnly') And '$(_WillCLRTestProjectBuild)'">
+    <MSBuild Projects="@(ProjectReference)" />
+    <MakeDir ContinueOnError="false" Directories="$(OutputPath)" />
```

Comment:
I don't remember the default behavior of this, but does the `MakeDir` task consider a directory that already exists an error?

Reply:
It continues if it already exists: the [docs](https://msdn.microsoft.com/en-us/library/s2448zz7.aspx) aren't clear, but looking at the [code](https://github.com/Microsoft/msbuild/blob/7d8c8da147252ea57f35d2c6ceee8e7c5ba458f5/src/Tasks/MakeDir.cs#L58) you can see it just does nothing if it's already there. 

 This is actually moved code from before, so the behavior should stay the same as previously.